src/internal/m365/collection/drive/collections_test.go

```diff
@@ -1477,7 +1477,7 @@ func (suite *OneDriveCollectionsUnitSuite) TestDeserializeMetadata() {
 					data.NoFetchRestoreCollection{Collection: mc}))
 			}
 
-			deltas, paths, canUsePreviousBackup, err := deserializeAndValidateMetadata(ctx, cols)
+			deltas, paths, canUsePreviousBackup, err := deserializeAndValidateMetadata(ctx, cols, fault.New(true))
```

Comment:
can you please expand this test to check that we actually add alerts when we think we should? Right now this test isn't really checking all that much since we no longer modify the input metadata